guide/14-deep-learning/faster-rcnn-object-detector.ipynb

```diff
@@ -0,0 +1,190 @@
+{
```

Comment:
<p>As we discussed, Please fix the last sentence.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/897/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>